src/pydocstyle/checker.py

```diff
@@ -104,7 +104,7 @@ class ConventionChecker:
         ".+"            # Followed by 1 or more characters - which is the docstring for the parameter
     )
 
-    def check_source(self, source, filename, ignore_decorators=None):
+    def check_source(self, source, filename, ignore_decorators=None, disable_skip_errors=False):
```

Comment:
Can we rename this `disable_skip_errors` has a certain cognitive dissonance. It almost feels like a double negative. Perhaps `ignore_inline_noqa` or `report_all_errors`?

Reply:
I agree with disliking the double negative. I had originally written this as `all_errors`, but reconsidered because elsewhere in the code the feature is referred to as `skip_errors` and I thought the consistency was valuable. I had also considered `skip_errors=True`, or `allow_skip_errors=True` but in general find that non-None/False defaults tend to cause problems, and it adds a few `not`s to the logic which makes it harder to read. `skip_errors`  could also imply a list of errors to skip.

I’m happy to change it, which do you prefer?